design-documents/api-clean-up/2.5/api-candidates.md

```diff
@@ -0,0 +1,1522 @@
+1. Klarna\Onsitemessaging\Block\Cart
+1. Magento\Backup\Block\Adminhtml\Backup
+1. Magento\InventoryShippingAdminUi\Block\Adminhtml\Shipment\Inventory
+1. Magento\InventoryShippingAdminUi\Block\Adminhtml\Order\View\ShipButton
+1. Magento\TwoFactorAuth\Block\ConfigureLater
+1. Magento\TwoFactorAuth\Block\Configure
+1. Magento\TwoFactorAuth\Block\Provider\U2fKey\Configure
+1. Magento\TwoFactorAuth\Block\Provider\U2fKey\Auth
+1. Yotpo\Yotpo\Block\Conversion
+1. Yotpo\Yotpo\Block\Yotpo
+1. Yotpo\Yotpo\Block\Adminhtml\Report\Reviews
+1. Magento\AdminNotification\Block\Window
+1. Magento\AdminNotification\Block\ToolbarEntry
+1. Magento\AdminNotification\Block\System\Messages\UnreadMessagePopup
+1. Magento\GoogleAdwords\Block\Code
+1. Magento\GoogleOptimizer\Block\Code\Product
+1. Magento\GoogleOptimizer\Block\Code\Category
+1. Magento\Marketplace\Block\Partners
+1. Magento\Multishipping\Block\Checkout\Addresses
+1. Magento\Multishipping\Block\Checkout\Results
+1. Magento\Multishipping\Block\Checkout\Overview
+1. Magento\Multishipping\Block\Checkout\Billing
+1. Magento\Multishipping\Block\Checkout\Success
+1. Magento\Multishipping\Block\Checkout\Link
+1. Magento\Multishipping\Block\Checkout\Shipping
+1. Magento\Multishipping\Block\Checkout\Billing\Items
+1. Magento\Multishipping\Block\Checkout\Address\Select
+1. Magento\PurchaseOrder\Block\Quote\Info\PurchaseOrder
+1. Magento\PurchaseOrder\Block\Order\Info\PurchaseOrder
+1. Magento\PurchaseOrder\Block\Link\PurchaseOrder
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Grid
+1. Magento\PurchaseOrder\Block\PurchaseOrder\View
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Success
+1. Magento\PurchaseOrder\Block\PurchaseOrder\History
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Totals
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Comments
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Items
+1. Magento\PurchaseOrder\Block\PurchaseOrder\EmailTotals
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Totals\Original
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Items\Messages
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\Buttons
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\NegotiableQuote
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\Shipping
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\Title
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\Order
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\Links
+1. Magento\PurchaseOrder\Block\PurchaseOrder\Info\Status
+1. Magento\PurchaseOrder\Block\Customer\Account\Link
+1. Magento\Sitemap\Block\Robots
+1. Magento\Ups\Block\Backend\System\CarrierConfig
+1. Magento\GoogleTagManager\Block\ListJson
+1. Magento\GoogleTagManager\Block\Ga
+1. Magento\GoogleTagManager\Block\Adminhtml\Ga
+1. Magento\GoogleTagManager\Block\Adminhtml\Creditmemo\Ga
+1. Magento\Support\Block\Adminhtml\Report\View\Tabs
+1. Magento\Support\Block\Adminhtml\Backup\Log
+1. Fastly\Cdn\Block\System\Config\Form\Modal\UploadVcl
+1. Fastly\Cdn\Block\Cache\Additional
```

Comment:
I believe Fastly\Cdn is not a VBE, is that extension added by mistake or we can influence it's vendor?

Reply:
I think it was added by mistake. @kandy pleas confirm

Reply:
yes, it's  3party extension, bundled with cloud meta package